src/main/java/pl/jakubowskir/timetable/trainer/controller/TrainerController.java

```diff
@@ -0,0 +1,26 @@
+package pl.jakubowskir.timetable.trainer.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import pl.jakubowskir.timetable.trainer.model.Trainer;
+import pl.jakubowskir.timetable.trainer.repository.TrainerRepository;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@RestController
+@AllArgsConstructor
+public class TrainerController {
+
+    protected static final AtomicInteger counter = new AtomicInteger(0);
+
+    private final TrainerRepository trainerRepository;
```

Comment:
To repository też

Reply:
@Gorgios faktyycznie xDD wcześniej bez tego błąd pokazywało, ale chyba dodawałem, jakiś import z trainerService